diff -Naur accountsservice-0.6.40.orig/data/org.freedesktop.accounts.policy.in accountsservice-0.6.40/data/org.freedesktop.accounts.policy.in
--- accountsservice-0.6.40.orig/data/org.freedesktop.accounts.policy.in	2014-11-04 10:46:00.000000000 +0700
+++ accountsservice-0.6.40/data/org.freedesktop.accounts.policy.in	2016-07-08 09:08:10.664617441 +0700
@@ -11,8 +11,8 @@
     <_description>Change your own user data</_description>
     <_message>Authentication is required to change your own user data</_message>
     <defaults>
-      <allow_any>auth_self</allow_any>
-      <allow_inactive>auth_self</allow_inactive>
+      <allow_any>yes</allow_any>
+      <allow_inactive>yes</allow_inactive>
       <allow_active>yes</allow_active>
     </defaults>
   </action>
