Index: accountsservice-0.6.40/data/org.freedesktop.accounts.policy.in
===================================================================
--- accountsservice-0.6.40.orig/data/org.freedesktop.accounts.policy.in
+++ accountsservice-0.6.40/data/org.freedesktop.accounts.policy.in
@@ -11,9 +11,7 @@
     <_description>Change your own user data</_description>
     <_message>Authentication is required to change your own user data</_message>
     <defaults>
-      <allow_any>auth_self</allow_any>
-      <allow_inactive>auth_self</allow_inactive>
-      <allow_active>yes</allow_active>
+      <allow_any>yes</allow_any>
     </defaults>
   </action>
 
